bld/namelist_files/use_cases/ssp370_cam6_noresm_tropstratchem.xml

```diff
@@ -0,0 +1,296 @@
+<?xml version="1.0"?>
+
+<namelist_defaults>
+
+<!--micro_do_sb_physics>   .true.   </micro_do_sb_physics-->
+<!--do_tms>                .false.  </do_tms-->
+<!--do_beljaars>           .true.   </do_beljaars-->
+
+<history_aerosol>.true.</history_aerosol>
+<modal_strat_sulfate>.true.</modal_strat_sulfate>
+<lght_no_prd_factor hgrid="0.9x1.25"> 1.0 </lght_no_prd_factor>
+<lght_no_prd_factor hgrid="1.9x2.5"> 10.0 </lght_no_prd_factor>
+
+<!-- Solar constant -->
+<solar_irrad_data_file>atm/cam/solar/SolarForcingCMIP6_18491230-22991231_c171031.nc</solar_irrad_data_file>
```

Comment:
Many files listed here are not in inputdata on Betzy (including this one). Does this run?
Note that most of the missing files seem to be looking for `0.9x1.25` while the directory has only `1.9x2.5` files.

Reply:
Hi Steve, you are right on this : indeed the 1.9x2.5 do not exist yet. 
I can generate them on ppi at some point in the future, and upload them to Betzy (once there is space).  Is it allowed to have them listed in the namelist anyway?  Should I add a comment in the code that people can contact me when they need the files (earlier).

Reply:
Hi Steve, I now added a comment in the file ssp370_cam6_noresm_tropstratchem.xml.  It says:
```xml
<!-- currently only emissions files at 1.9x2.5 resolution exist.  Files at 0.9x1.25 resolution can be generated on request -->
```